src/Snppts/Snppts.csproj

```diff
@@ -14,7 +14,6 @@
 
   <ItemGroup>
     <Folder Include="Views\Snippets\" />
-    <Folder Include="Snippets\" />

```

Comment:
Any idea why this got excluded?

Reply:
I see it included.

Reply:
Simply add the FoodApp.cs class.

Reply:
This part is solved.

Reply:
Hm not exactly, I still see a line being deleted here